pom.xml

```diff
@@ -56,7 +56,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.0</version>
+            <!-- needed to avoid ClassNotFoundException for PipelineTriggersJobProperty in tests -->
+            <version>2.4</version>
```

Comment:
Trying to execute the declarative pipeline in the test would fail with CNFE specified in the comment. Checking the history, this is earliest version that introduces this class. I wanted to specify this with scope=test, but it seems that Maven 3 doesn't handle the same dep being declare with different scopes. Is there a better way to handle this?

Reply:
It's okay to bump the base dependency -- people will kick up a fuss if you're pulling in something dicey

Reply:
Left bumped dep version and removed comment in commit a42a59e